src/test/codegen/thread-local.rs

```diff
@@ -19,15 +19,15 @@ thread_local!(static A: Cell<u32> = const { Cell::new(1) });
 // CHECK-LABEL: @get
 #[no_mangle]
 fn get() -> u32 {
-    // CHECK: %0 = load i32, i32* bitcast ({{.*}} [[TLS]] to i32*)
+    // CHECK: %0 = load i32, i32* [[TLS]], align 4
```

Comment:
I think doing something like this should be fine:
```suggestion
    // CHECK: %0 = load i32, i32* {{.*}} [[TLS]]
```

Reply:
Thanks! Updated to work with current Rust LLVM and HEAD LLVM.